theme/highcontrast.less

```diff
@@ -168,8 +179,10 @@
         }
     }
 
+    rect.blocklyFlyoutButtonBackground,
```

Comment:
Is there a case of there being a `blocklyFlyoutButtonBackground` that isn't a rect? I think should be fine to just add rect without duplicating selector here.